### PR TITLE
feat(CostModelWizard): 3145 distribution step (crawl)

### PR DESCRIPTION
**Pull request by gitdallas**
closes: https://issues.redhat.com/browse/COST-3145

### CODE REVIEW

**Review by dlabrecq — COMMENTED** [review]
Has the backend pushed support to stage and/or prod? I noticed the cost-model request didn't error with the new properties.

If the backend isn't ready, you could wrap this feature in an Unleash toggle? That way, you can merge your code to main.

File: src/locales/messages.ts [hunk 1]
```diff
@@ -1461,6 +1495,12 @@ export default defineMessages({
       'https://access.redhat.com/documentation/en-us/cost_management_service/2022/html-single/using_cost_models/index#cost-model-terminology',
     id: 'docsCostModelTerminology',
   },
+  docsCostModelsDistribution: {
+    defaultMessage:
+      'https://access.redhat.com/documentation/en-us/cost_management_service/2022/html/using_cost_models/assembly-setting-up-cost-models#creating-an-AWS-Azure-cost-model_setting-up-cost-models',
```
[the remark this answers is not in the document]
Comment (gitdallas) [reply src/locales/messages.ts:1500]: the sketches currently have the same URL listed in both places https://www.sketch.com/s/cc6b3d35-a41c-4810-bdd8-4f70a6a3e4e5/a/AxvKMPr

File: src/routes/costModels/createCostModelWizard/wizard.styles.ts [hunk 2]
```diff
@@ -1,4 +1,7 @@
+import { styles as costCalcStyles } from 'routes/costModels/costModel/costCalc.styles';
+
 export const styles = {
+  ...costCalcStyles,
```
[the remark this answers is not in the document]
Comment (dlabrecq) [reply src/routes/costModels/createCostModelWizard/wizard.styles.ts:4]: Yes, you could do that

**Comment by gitdallas** [comment]
@dlabrecq - looking at putting this behind unlease... i've kinda had to move things around a bit and if i have this as a flag, i'm not 100% sure how to go about it. would i keep the new step and just remove the two new options?

File: src/locales/messages.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (gitdallas) [reply src/locales/messages.ts:1500]: I did add a new placeholder... same url but using a unique intl message so when we get a new url we can just replace it there.

**Review by dlabrecq — APPROVED** [review]

**Pull request merged**